pkg/ca-certificates/Makefile

```diff
@@ -1,19 +1,17 @@
 .PHONY: tag push
 
-BASE=alpine:3.5
 IMAGE=ca-certificates
 
-SHA_IMAGE=alpine:3.5@sha256:dfbd4a3a8ebca874ebd2474f044a0b33600d4523d03b0df76e5c5986cb02d7e8
+BASE=$(shell cat Dockerfile | grep '^FROM' | head -1 | sed -e 's/FROM //' -e 's/ as .*//')
+
+HASH?=$(shell git ls-tree HEAD -- ../$(notdir $(CURDIR)) | awk '{print $$3}')
 
 default: push
 
 hash: Dockerfile
-	DOCKER_CONTENT_TRUST=1 docker pull $(BASE)
-	tar cf - $^ | docker build --no-cache -t $(IMAGE):build -
-	CONTAINER=$$(docker create $(IMAGE):build /dev/null); \
-		mkdir tmp && docker export $${CONTAINER} | tar xf - -C tmp && \
-		cat tmp/etc/ssl/certs/ca-certificates.crt | docker run --rm -i $(SHA_IMAGE) sha1sum - | sed 's/ .*//' > $@ && \
-		rm -rf tmp && docker rm $${CONTAINER}
+	docker pull $(BASE)
+	tar cf - $^ | docker build --network=none -t $(IMAGE):build -
+	echo $(HASH) > $@
```

Comment:
These three steps should not be necessary anymore as we use the git tree hash as the hash. So there is no need  for the `hash` step. In the `./kernel` I just have a `build` target which does a `docker build -t $(IMAGE):$(HASH)` conditional on a `pull` of `$(IMAGE):$(HASH)` failing, but I added that to be able to build and test kernels locally without pushing them.